src/Item.js

```diff
@@ -83,6 +142,25 @@ function Item({ userToken, item }) {
         />
         {itemName}
       </label>
+      {/* // [delete-item] 1. Create a button next to each item in list */}
+      <button type="button" onClick={handleOpenModal}>
+        X
+      </button>
+      {/* // [delete-item] 2. Create a button modal that will render when user tries to delete item */}
+      <Modal
+        open={openModal}
+        onClose={handleCloseModal}
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+      >
+        <div>
+          <StyledModal>
+            <h1>Are you sure you want to delete this item?</h1>
```

Comment:
Looks like you need to add an `id="simple-modal-title"` to complete the `aria-labelledby` connection.

Reply:
Oh good point!